api/etcdserverpb/rpc.pb.go

```diff
@@ -1792,7 +1792,9 @@ type SnapshotResponse struct {
 	// remaining_bytes is the number of blob bytes to be sent after this message
 	RemainingBytes uint64 `protobuf:"varint,2,opt,name=remaining_bytes,json=remainingBytes,proto3" json:"remaining_bytes,omitempty"`
 	// blob contains the next chunk of the snapshot in the snapshot stream.
-	Blob                 []byte   `protobuf:"bytes,3,opt,name=blob,proto3" json:"blob,omitempty"`
+	Blob []byte `protobuf:"bytes,3,opt,name=blob,proto3" json:"blob,omitempty"`
+	// version of server that created the snapshot.
+	Version              string   `protobuf:"bytes,4,opt,name=version,proto3" json:"version,omitempty"`
```

Comment:
Note: This is streaming API. I assume we will send the version in all the chunks.

Reply:
Yes, decided it would make most sense to store version in each chunk. It should not have big impact of proto size as size(.Version) << size(.Blob).

Reply:
SG. Please just add it to the comment.